skyvern/forge/sdk/api/llm/schema_validator.py

```
@@ -0,0 +1,236 @@
+        schema: The JSON schema to validate
+
+    Raises:
+        InvalidSchemaError: If the schema is invalid with details about why
+    """
+    if schema is None or isinstance(schema, (str, list)):
+        return
+
+    try:
+        Draft202012Validator.check_schema(schema)
+    except SchemaError as e:
+        error_message = f"Invalid JSON schema: {str(e)}"
```

Ruff (0.14.5)

Use explicit conversion flag

Replace with conversion flag

(RUF010)